javascriptv3/example_code/libs/logger.js

```diff
@@ -21,17 +23,20 @@ export class Logger {
     if (!message) {
       console.log("\n", "*".repeat(80), "\n");
     } else {
-      console.log(
-        "\n",
-        "*".repeat(80),
-        "\n",
-        "** ",
-        message,
-        " ".repeat(80 - message.length - 8),
-        "**\n",
-        "*".repeat(80),
-        "\n",
-      );
+      const maxLengthPerLine = 74;
```

Comment:
Magic number - add a comment about why it's 74 (80 - ((2 + 2) + 2))

Reply:
Updated.